libnavigation-base/src/main/java/com/mapbox/navigation/base/options/NavigationOptions.kt

```diff
@@ -40,10 +46,14 @@ class NavigationOptions private constructor(
         fun onboardRouterConfig(onboardRouterConfig: MapboxOnboardRouterConfig) =
             apply { this.onboardRouterConfig }
```

Comment:
missed 
```suggestion
            apply { this.onboardRouterConfig = onboardRouterConfig }
```

Reply:
need tests to avoid these type of issues

Reply:
👍 I'll create them